packages/perspective/webpack-plugin.js

```diff
@@ -7,84 +7,61 @@
+const BLOB_LOADER = require.resolve("./loader/blob_worker_loader.js");
 
-const BABEL_CONFIG = require("./babel.config.js");
+const include = [__dirname];
 
 class PerspectiveWebpackPlugin {
     constructor(options = {}) {
         this.options = options;
     }
 
     apply(compiler) {
-        const load_path = [__dirname];
         const rules = [
             {
-                test: /\.less$/,
-                exclude: /themes/,
-                include: load_path,
-                use: [{loader: "css-loader"}, {loader: "clean-css-loader", options: {level: 2}}, {loader: "less-loader"}]
-            },
-            {
-                test: /\.(html)$/,
-                include: load_path,
-                use: {
-                    loader: "html-loader",
-                    options: {}
-                }
-            },
-            {
-                test: /\.(arrow)$/,
-                include: load_path,
+                test: /psp\.(sync|async)\.wasm\.js$/,
+                include,
                 use: {
-                    loader: "arraybuffer-loader",
-                    options: {}
+                    loader: CROSS_ORIGIN_FILE_LOADER, 
+                    options: {
+                        name: "[name]"
+                    }
                 }
             }
         ];
 
         if (this.options.build_worker) {
             rules.push({
                 test: /perspective\.(asmjs|wasm)\.js$/,
-                include: load_path,
+                include,
                 use: [
                     {
-                        loader: WORKER_LOADER_PATH,
+                        loader: FILE_WORKER_LOADER,
                         options: {name: "[name].js", compiled: true}
                     },
                     {
-                        loader: BLOB_LOADER_PATH,
-                        options: {name: "[name].worker.js"}
+                        loader: BLOB_LOADER,
+                        options: {
+                            name: "[name].worker.js"
+                        }
                     }
                 ]
             });
         } else {
             rules.push({
                 test: /perspective\.(wasm|asmjs)\.js$/,
-                include: load_path,
+                include,
                 use: {
-                    loader: WORKER_LOADER_PATH,
-                    options: {name: "[name].js"}
+                    loader: FILE_WORKER_LOADER,
+                    options: {
+                        name: "[name].js"
+                    }
                 }
             });
         }
 
-        rules.push({
-            test: /\.js$/,
-            include: load_path,
-            exclude: /node_modules[/\\](?!\@jpmorganchase)|psp\.(asmjs|async|sync)\.js|perspective\.(asmjs|wasm)\.worker\.js/,
-            loader: "babel-loader",
-            options: BABEL_CONFIG
-        });
-
-        rules.push({
```

Comment:
I think removing this breaks the `"inline"` plugin flag, which needs to work in concert with this specific loader, as in the Jupyter plugin, but there are other reasons to use it - e.g. when we have webpack 4 support where [wasm loading from emscripten modules is completely ****ing broken](https://github.com/webpack/webpack/issues/7352).